ops/pipeline/get-image-tag.sh

```diff
@@ -1,4 +1,4 @@
 ## Update the following line to test changes to CI images
 ## See https://xgboost.readthedocs.io/en/latest/contrib/ci.html#making-changes-to-ci-containers
 
-IMAGE_TAG=main
+IMAGE_TAG=PR-14
```

Comment:
Using CI images from https://github.com/dmlc/xgboost-devops/pull/14 to test the PR with latest Rapids and Dask